### PR TITLE
refactor(ui/views): normalization

### PR DESCRIPTION
**Pull request by 121watts**

### CODE REVIEW

File: ui/src/dashboards/components/Dashboard.tsx [hunk 1]
```diff
@@ -46,24 +47,34 @@ class DashboardComponent extends PureComponent<Props> {
...
+        <SpinnerContainer
+          loading={viewsStatus}
+          spinnerComponent={<TechnoSpinner />}
+        >
+          <Page.Contents
+            fullWidth={true}
+            scrollable={true}
+            className="dashboard"
+          >
+            {!!cells.length ? (
```
Comment (ebb-tide) [inline ui/src/dashboards/components/Dashboard.tsx]: perhaps this should check whether views have loaded? cells array sometimes comes back as [undefined, undefined...]

File: ui/src/dashboards/actions/thunks.ts [hunk 2]
```diff
@@ -261,9 +263,9 @@ export const getDashboard = (dashboardID: string) => async (
   getState: GetState
 ): Promise<void> => {
   try {
-    // Fetch the dashboard and all variables a user has access to
+    // Fetch the dashboard, views, and all variables a user has access to
```
Comment (ebb-tide) [inline ui/src/dashboards/actions/thunks.ts]: dispatch(creators.setDashboard(dashboardID, RemoteDataState.Loading))

Comment (121watts) [reply ui/src/dashboards/actions/thunks.ts]: thanks @ebb-tide !!!

File: ui/src/shared/components/cells/Cell.tsx [hunk 3]
```diff
@@ -85,12 +84,12 @@ class CellComponent extends Component<Props, State> {
...
 
   private get view(): JSX.Element {
-    const {timeRange, manualRefresh, view, viewsStatus} = this.props
+    const {timeRange, manualRefresh, view} = this.props
 
     return (
       <SpinnerContainer
-        loading={viewsStatus}
-        spinnerComponent={<EmptyGraphMessage message="Loading..." />}
+        loading={view.status || RemoteDataState.Loading}
```
Comment (ebb-tide) [inline ui/src/shared/components/cells/Cell.tsx]: it makes me nervous that you need this here

File: ui/src/types/index.ts [hunk 4]
```diff
@@ -1,27 +1,37 @@
+export * from './alerting'
```
Comment (ebb-tide) [inline ui/src/types/index.ts]: alphabetization is nice thank you...

**Pull request merged**